google/cloud/bigquery/client.py

```diff
@@ -564,8 +567,8 @@ def _ensure_bqstorage_client(
             A BigQuery Storage API client.
         """
         try:
-            from google.cloud import bigquery_storage  # type: ignore
-        except ImportError:
+            bigquery_storage = BQ_STORAGE_VERSIONS.try_import()
+        except BigQueryStorageNotFoundError:
```

Comment:
If we implement `try_import(raise_if_error=True)`, I think we can combine this try/except block and the one where we call `verify_version()` into one.

Reply:
Done, change made at #1699